src/tokens/ERC721.sol

```diff
@@ -28,7 +28,9 @@ abstract contract ERC721 {
                             ERC-721 STORAGE
     //////////////////////////////////////////////////////////////*/
     
-    uint256 public totalSupply;
+    uint256 public minted;
+
+    uint256 public burned;
```

Comment:
why use two seperate vars here? think i'd prefer to keep one totalSupply but willing to have my mind changed